model/host/host_test.go

```diff
@@ -843,7 +843,7 @@ func TestHostElapsedCommTime(t *testing.T) {
 }
 
 func TestHostUpsert(t *testing.T) {
-	assert := assert.New(t)
+	assert := assert.New(t) // nolint
```

Comment:
Is this sustainable? Adding nolint to each new test that uses assert seems like a hassle